documentation/user_docs/docs/manual/Extracting_data_from_SQW_files_and_objects.rst

```diff
@@ -0,0 +1,637 @@
+``dnd`` file and converts it to an object of the same or reduced size and dimensions.
+The resulting cut object is itself an ``sqw`` or ``dnd`` object which can be plotted, manipulated,
+etc. like any other ``sqw`` or ``dnd`` object. If the resulting object is itself too big to fit memory,
+it will be backed by appropriate ``sqw`` file. 
+The required inputs are as follows:
+
+.. note::
+
+   For the differences between ``dnd`` and ``sqw`` objects see: :ref:`here
+   <manual/FAQ:What is the difference between sqw and dnd objects>`
+
+.. code-block:: matlab
+
+   my_cut = cut(data_source, proj, p1_bin, p2_bin, p3_bin, p4_bin,varargin)
+
+where ``varargin`` describes optional arguments and may contain:
+
+  ``-nopix``   -- the key requests that the cut should contain just image (``dnd`` object) and drop all pixels contributing into this image. This  option is faster but should not be used if you want to do subsequent cuts from the resulting cut ``my_cut``.
+  
+  ``filename`` -- if provided, requests that the resulting file is backed by the file with the ``filename``. This option is mandatory if cut is called without output argument.
+
+Cutting consists of a rebinning of the pixels into the bins specified by the cut
+parameters (described below).  These binned pixels will make up the ``dnd`` of
+the output object which contains information regarding the plottable data.
+
+
+Data Source
+-----------
+
+``data_source`` in ``cut`` is either a string giving the full filename (including path) of
+the input ``.sqw`` file or just the variable containing an ``sqw`` or ``dnd``
+object stored in memory from which the pixels will be taken.
+
+.. warning::
+
+   By convention both ``sqw`` and ``dnd`` objects use the ``.sqw`` suffix when
+   stored to disc. It is advisable to name the file appropriately to distinguish
+   the types stored inside, e.g. ``MnO2_sqw.sqw``, ``MnO2_d2d.sqw``.
+
+Projection (proj)
+-----------------
+
+Second argument defines the coordinate system and thus the meaning of the 
+`Binning arguments`_ you provided as arguments to `cut` and will 
+use to plot and analyse the data.
+
+``proj`` should be an child of a ``aProjectionBase`` class (such as ``line_proj``,
+``sphere_proj``, etc.) containing information about the the coordinate
+system you wish to use to plot and analyse the data.
+
+.. note::
+
+   Because each point of ``sqw.pix`` data describes the
+   position in the reciprocal space and energy transfer, the underlying pixels will be
+   unchanged. For image (``dnd`` object) it is possible to redefine the coordinate system with the one of
+   your choice; the projection and binning parameters merely describe how pixels will be accumulated
+   (binned) and thus displayed in image coordinate system.
+   
+Different projections in more details are covered in `Projection in more details`_ chapter below.
+
+
+Binning arguments
+-----------------
+
+.. _barguments:
```

Comment:
You define a reference which is never used

Reply:
That's your reference BTW. I've removed mine, so probably can this too.

Reply:
actually, it should be here

Reply:
or maby be can be removed. Dodgy `rst`